backend/plugin/db/pg/pg.go

```diff
@@ -239,11 +239,9 @@ func (driver *Driver) getDatabases(ctx context.Context) ([]*storepb.DatabaseSche
 
 // getVersion gets the version of Postgres server.
 func (driver *Driver) getVersion(ctx context.Context) (string, error) {
-	// SHOW server_version_num returns an integer such as 100005, which means 10.0.5.
-	// It is more convenient to use SHOW server_version to get the version string.
-	// PostgreSQL supports it since 8.2.
+	// PostgreSQL supports SHOW server_version since 8.2.
 	// https://www.postgresql.org/docs/current/functions-info.html
-	query := "SHOW server_version_num"
+	query := "SHOW server_version"
```

Comment:
We probably don't want to show the string "14.9 (Homebrew)".

Reply:
I get plain `15.1`. Do you start a pg instance installed by homebrew?
![image](https://github.com/bytebase/bytebase/assets/71714656/f47afd74-6c25-4986-ad5f-06294e0c16a0)

Reply:
Yup. The could vary by Cloud providers. If server_version_num is a consistent format within Postgres, let's construct the semantic version from server_version_num instead.

Reply:
pg offical doc recommands using `server_version_num` instead of parsing the text version. Maybe `server_version_num` is better.
https://www.postgresql.org/docs/current/functions-info.html